lib/repl.js

```diff
@@ -772,7 +772,7 @@ function complete(line, callback) {
         name = files[f];
         ext = path.extname(name);
         base = name.slice(0, -ext.length);
-        if (base.match(/-\d+\.\d+(\.\d+)?/) || name === '.npm') {
+        if (base.match(/-\d+\.\d+/) || name === '.npm') {
```

Comment:
Also may as well change this to a `regex.test()` since the return value is discarded.